sycl/unittests/scheduler/EnqueueWithDependsOnDeps.cpp

```diff
@@ -159,7 +159,7 @@ class DependsOnTests : public ::testing::Test {
 
   std::mutex m;
   std::function<void()> CustomHostLambda = [&]() {
-    std::unique_lock InsideHostTaskLock(this->m);
+    std::unique_lock<std::mutex> InsideHostTaskLock(this->m);
```

Comment:
@KseniyaTikhomirova , this `this->m` looks suspicious given the `m` on line 160. Please review post-commit.